features/overflow-clip-margin.yml

```
@@ -0,0 +1,9 @@
+name: Overflow clip margin
+description: "The `overflow-clip-margin` CSS property sets how far overflow content will display outside the bounds of an element when clipped by effects such as `overflow: clip`."
```

Obscure style nit: avoid the future tense

```suggestion
description: "The `overflow-clip-margin` CSS property sets how far overflow content may appear outside the bounds of an element before it's clipped by effects such as `overflow: clip`."
```